.github/workflows/ofrak-type-ci.yml

```diff
@@ -0,0 +1,619 @@
+        run: |
+          pip install dist-patch-maker/*.tar.gz
+
+      - name: Run OFRAK Patch Maker tests (Linux/macOS)
+        if: runner.os != 'Windows'
+        run: |
+          cd ofrak_patch_maker
+          make test
+
+      - name: Run OFRAK Patch Maker tests (Windows)
+        if: runner.os == 'Windows'
+        run: |
+          cd ofrak_patch_maker
+          make test PYTHON=python
+
+  build-ofrak-wheels:
+    name: Build OFRAK wheels on ${{ matrix.os }} for Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          lfs: true
+
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+
+      - name: Build frontend
+        run: |
+          if [ -d "frontend" ]; then
+            cd frontend
+            npm install
+            npm run build
+            cd ../ofrak_core
+            cp -r ../frontend/dist src/ofrak/gui/public
+          else
+            echo "No frontend directory found, creating empty public directory"
+            mkdir -p ofrak_core/src/ofrak/gui/public
+          fi
+        shell: bash
+
+      - name: Install build tools
+        run: |
+          python -m pip install -r pip-requirements.txt
+          pip install cibuildwheel
+
+      - name: Set Python version for cibuildwheel
+        id: py-version
+        run: |
+          PY_VERSION="${{ matrix.python-version }}"
+          PY_VERSION_NO_DOT=${PY_VERSION//./}
+          echo "version=${PY_VERSION_NO_DOT}" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: Build OFRAK wheels with cibuildwheel
+        env:
+          CIBW_BUILD: "cp${{ steps.py-version.outputs.version }}-*"
+          CIBW_SKIP: "*-win32 *-manylinux_i686 *-musllinux*"
+          CIBW_ARCHS_WINDOWS: "AMD64"  # Simplified for now
+          CIBW_ARCHS_MACOS: "x86_64 arm64"
+          CIBW_ARCHS_LINUX: "x86_64"  # Simplified for now
+          CIBW_BUILD_VERBOSITY: 1
+          CIBW_BEFORE_BUILD: "pip install setuptools wheel"
+        run: |
+          cd ofrak_core
+          cibuildwheel --output-dir ../wheelhouse
+        shell: bash
+
+      - name: List built wheels
+        run: |
+          echo "Built OFRAK wheels:"
+          ls -la wheelhouse/ || echo "No wheelhouse found"
+        shell: bash
+
+      - name: Upload wheels
+        uses: actions/upload-artifact@v4
+        with:
+          name: ofrak-wheels-${{ matrix.os }}-py${{ matrix.python-version }}
+          path: wheelhouse/
+          retention-days: 30
```

Comment:
This is duplicated in the other YAML file. Which is the one that actually gets used?